details_energy.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: index
+title: The GCAM Water Model
+prev: supply_energy.html
+next: supply_energy.html
+gcam-version: v5.3 
+---
+
+This page provides more detailed explanations of the descriptions provided in the [Energy Supply](supply_energy.html) and [Energy Demand](demand_energy.html) modeling pages.
+
+{:toc}
```

Comment:
{:toc} renders as is (on GitHub).  I assume it's supposed to be doing something else.